src/backend/cdb/cdbmutate.c

```diff
@@ -75,8 +75,19 @@ typedef struct ApplyMotionState
 								 * plan_tree_walker/mutator */
 	int			nextMotionID;
 	int			sliceDepth;
-	bool		containMotionNodes;
+	bool		containMotionNodes;	/* Obsolete and unused, see below. Kept for
+									 * ABI compatability. */
```

Comment:
do we need to maintain an ABI for a private structure? It doesn't seem to be available to extensions.

Reply:
I don't think so. Just trying to check why ABI tests suddenly exploded.

Reply:
it seems that the reason is that the default branch has been updated and adb-6.x-dev has not synchronized the changes yet. I don't think it requires any action on your part. Take it back the way it was.